diff --git a/node_modules/@nguniversal/builders/src/prerender/index.js b/node_modules/@nguniversal/builders/src/prerender/index.js
index 9e8d19c..8c7612b 100755
--- a/node_modules/@nguniversal/builders/src/prerender/index.js
+++ b/node_modules/@nguniversal/builders/src/prerender/index.js
@@ -139,7 +139,7 @@ async function _renderUniversal(routes, context, browserResult, serverResult, br
             if (browserOptions.serviceWorker) {
                 spinner.start('Generating service worker...');
                 try {
-                    await (0, service_worker_1.augmentAppWithServiceWorker)(projectRoot, (0, core_1.normalize)(outputPath), browserOptions.baseHref || '/', browserOptions.ngswConfigPath);
+                    await (0, service_worker_1.augmentAppWithServiceWorker)(projectRoot, context.workspaceRoot, (0, core_1.normalize)(outputPath), browserOptions.baseHref || '/', browserOptions.ngswConfigPath);
                 }
                 catch (error) {
                     spinner.fail('Service worker generation failed.');
